web/js/containers/toolbar.js

```diff
@@ -30,17 +30,15 @@ const CUSTOM_MODAL_PROPS = {
   TOOLBAR_PROJECTION: {
     headerText: null,
     type: 'toolbar',
-    modalClassName: 'toolbar-list-modal toolbar-modal',
+    modalClassName: 'toolbar-list-modal toolbar-projeciton-modal toolbar-modal',
```

Comment:
misspelling of `projection`

Reply:
Good catch. Updated the spelling in `toolbar.js` and `toolbar.css`